pkgs/by-name/cl/cloud-ddns/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "cloud-ddns";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "jason-m";
+    repo = pname;
```

```suggestion
    repo = "cloud-ddns";
```
Do not rely on programs and repos being named the same (see Catppuccin as an antiexample)